src/index.html

```diff
@@ -4,9 +4,461 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Kosugi&family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog">
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              class="nav__link is-active nav__link--noMargin"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+            samsung
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+            smartphones
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops&computers"
+            >
+            laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#gadgets"
+            >
+            gadgets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#tablets"
+            >
+            tablets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#photo"
+            >
+            photo
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#video"
+            >
+            video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog__section">
```

Comment:
In my opinion, in this case, it should be "catalog" instead of the body element, as in the body we also have navigation and other staff.
And inside of main, you can create a section - "catalog__section"